test/src/test/java/org/corfudb/runtime/concurrent/CorfuQueueTxTest.java

```diff
@@ -24,7 +25,22 @@
 @Slf4j
 public class CorfuQueueTxTest extends AbstractTransactionsTest {
     @Override
-    public void TXBegin() { OptimisticTXBegin(); }
+    public void TXBegin() {
+        OptimisticTXBegin();
```

Comment:
How about TXBegin() calling TXBegin(TransactionType.OPTIMISTIC)? Maybe it is more intuitive that way to implement default parameters.

Reply:
Sounds good. Done.